### PR TITLE
YARA Indicator

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10275

## Description
This is the indicator type, fields and layout for the YARA indicator type.

New indicator type:
- YARA Rule

New indicator fields:
- Author
- Raw Rule
- Rule Condition
- Rule ID
- Rule Reference
- Rule Strings

New Layout:
- YARA Rule Layout

### CODE REVIEW

File: Packs/Yara/IndicatorFields/indicatorfield-Author.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
```
Comment (altmannyarden) [inline Packs/Yara/IndicatorFields/indicatorfield-Author.json:21]: if this field associated to all, dont you think we should add the field to commontypes pack?

File: Packs/Yara/Layouts/layoutscontainer-YARA_Rule_Layout.json [hunk 2]
```diff
@@ -74,7 +74,7 @@
         ]
     },
     "group": "indicator",
-    "id": "YARA Rule Layout",
+    "id": "YARA Rule Layout_copy",
```
Comment (altmannyarden) [inline Packs/Yara/Layouts/layoutscontainer-YARA_Rule_Layout.json:77]: remove the copy

File: Packs/Yara/Layouts/layoutscontainer-YARA_Rule_Layout.json [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+                        "fieldId": "indicator_rulecondition",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "indicator_rawrule",
+                        "isVisible": true
+                    }
+                ],
+                "isVisible": true,
+                "name": "Custom Fields",
```
Comment (altmannyarden) [inline Packs/Yara/Layouts/layoutscontainer-YARA_Rule_Layout.json:68]: I dont see any section with this name

File: Packs/Yara/IndicatorFields/indicatorfield-Author.json (hunk 1 above)
Comment (Ni-Knight) [reply Packs/Yara/IndicatorFields/indicatorfield-Author.json:21]: Done

File: Packs/Yara/Layouts/layoutscontainer-YARA_Rule_Layout.json (hunk 2 above)
Comment (Ni-Knight) [reply Packs/Yara/Layouts/layoutscontainer-YARA_Rule_Layout.json]: Done

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**